packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -296,10 +294,10 @@ const SigninContainer = ({
                 password,
                 clientSalt:
                   credentialStatusData.data?.credentialStatus.clientSalt ||
-                  (await createSaltV2()),
```

Comment:
I removed the `await` from this line and the one setting `kB` because these aren't async functions.